vivisect/vivbin.py

```diff
@@ -60,6 +64,22 @@ def main():
     level = e_common.LOG_LEVELS[vw.verbose]
     e_common.initLogging(logger, level=level)
 
+    if args.server_mode:
+        import vivisect.remote.server as vr_server
+        if len(args.file):
+            file = args.file[-1]
```

Comment:
Wait. This might actually be a bug. Your `--server` definition on line 40 says to `store_true` but you're treating it here like a filename? Something seems off about that.

Reply:
am i?  i'm using the positional arguments (see the description on `--server`)

Reply:
Huh. Yea I missed that. Cool.